js/src/lib/interfaces/Types.ts

```diff
@@ -225,6 +225,7 @@ export interface ModelData {
 		inference?: boolean | {
 			parameters?: Record<string, any>;
 		};
+		widget_realtime_asr?: boolean;
```

Comment:
The idea is to show realtime widget feature only on models that enabled this flag

cc: @julien-c @osanseviero